test/jdk/java/net/httpclient/ResponseBodyBeforeError.java

```diff
@@ -346,7 +346,9 @@ protected ServerSocket newServerSocket() throws IOException {
 
         @Override
         public void run() {
+            int maxUnexpected = 2;
```

Comment:
Hello Daniel, should we have this `maxUnexpected` at all? Perhaps, we keep accepting and ignoring connections/requests that don't match our expectations? The reason I ask this is, I don't think we will know the exact number of unexpected requests/connections that this server might receive when this test is going on, so with this value of 2, I guess there still would be a chance that this test fails because of those unexpected connections?

Reply:
I have increase the limit to 10